.github/workflows/tests.yml

```diff
@@ -49,18 +51,18 @@ jobs:
             runs-on: ubuntu-22.04
             fuse_package: libfuse3-dev
           # fuse-t
-          - cc: gcc-13
-            cxx: g++-13
+          - cc: gcc
+            cxx: g++
             clang_major_version: null
             clang_repo_suffix: null
-            runs-on: macos-13
+            runs-on: macos-15
             fuse_package: fuse-t
           # macFUSE
-          - cc: clang-15
-            cxx: clang++-15
-            clang_major_version: 15
+          - cc: clang
+            cxx: clang++
+            clang_major_version: null
             clang_repo_suffix: null
-            runs-on: macos-13
+            runs-on: macos-15
```

Comment:
@mpartel I would like to understand the idea behind updating to macos-15. Does this fix parts of compilation (and if so: which parts) or is it housekeeping? If housekeeping, my vote, for splitting it of the fixing pull request.

Reply:
Housekeeping / keeping up with the latest macos, which occasionally breaks things. Normally I'd be all for small PRs but this is a very low-activity project and I have quite little time, so I'm taking shortcuts in this case.